### ISSUE TITLE
Still more errors

### ISSUE THREAD

**Issue by DenisCarriere** [sharedstreets/sharedstreets-explorer#5]
@schnerd from a fresh install, does it still work for you?

I'm getting more React related errors.

![image](https://user-images.githubusercontent.com/550895/36065155-c36701c2-0e64-11e8-826b-6f1ce184569a.png)

**Comment by kpwebb** [comment]
Let's regroup on the UI -- we've created an updated version of the UI based on @DenisCarriere new JS libraries, and it might be better to just work off that common base, rather than have two react-based front-ends going in parallel. This new UI also has the concept of a "tiled data source" and dynamically loads/renders data as the user move the map. 

I'd like to split the UI into two distinct projects that draw from a common set of libraries for the UI and map overlays.

1) A barebones locally run-able CLI tool for viewing SS data files ("$: ss-explorer --path [ss_data_files]" kicks off a local server that can be opened in browser to view data)

2) The full-blown "ss-ui" which provides a hosted endpoint for the data viewer and is designed to work with hosted global data sets

Both could use our plug-in architecture for loading custom overlays. At the moment I'm working on a dynamic loader for pulling down code to render various data formats. I think it's possible to include that in the CLI (the browser would just grab hosted plugin packages), but at minimum the CLI tool could view the core SS data types viewable. 

Next step here I need to tie up loose ends on the UI code (working on that now!). And let's work together to break it apart so we can build both 1) and 2) off the same shared code base.

**Comment by schnerd** [comment]
@DenisCarriere yeah I saw those too and came to the conclusion that it was some faulty code in [antd](https://ant.design/docs/react/introduce).

kevin that plan sounds good, will keep my eyes peeled for a new ss-ui repo 🙂

**Comment by DenisCarriere** [comment]
👍 I'll keep focusing on building up some "core" JS modules that have full coverage testing, that way whenever do we use them into the UI repo we have less and less errors to fix.

We should look into building the UI platform also in Typescript, it's a pain at first, but the error checking is very valuable in the long run.
